deployment/k8s/charts/aggregates/migrationAssistantWithArgo/values.yaml

```diff
@@ -47,12 +47,23 @@ images:
     tag: latest
     pullPolicy: IfNotPresent
 
-snapshotBucketConfiguration:
-  create: true
-  bucketName: snapshot
+# Default AWS values are dummy values for LocalStack usage
+aws:
+  eksEnabled: false
   region: us-east-2
+  account: "123456789012"
+
+stageName: dev
+
+defaultBucketConfiguration:
+  create: true
+  deleteOnUninstall: true
+  emptyBeforeDelete: true
```

Comment:
The questions about how are values settings preserved are relevant here.  If a user changes their mind after an installation, will modified values here have an impact or not?  It could be helpful to add a comment here or in the Chart's README to explain that detail (& explain that it's really up to helm)